### DISCUSSION TITLE
Can the acmebot terraform module be combined with the acme provider?

### DISCUSSION THREAD

**Discussion by Kralizek** [shibayan/terraform-azurerm-keyvault-acmebot#63] [Q&A]
I read from the documentation that acmebot takes care of renewing automatically certificates added via their interface.

This requires manual work. Ideally, I'd love to use the [acme provider](https://registry.terraform.io/providers/xaevman/acme/latest) to create new certificates and delegate to acmebot their renewal.

Is this scenario supported?

The question is: is the KV monitored for any certificate, or only the certificates added via the UI are actually managed?

**Comment by shibayan** [comment]
Acmebot uses the tags on Key Vault certificates to determine whether they are under Acmebot's control, so even certificates issued by other methods are treated as renewal targets if they have been given the appropriate tags.

- https://github.com/shibayan/keyvault-acmebot/blob/master/KeyVault.Acmebot/Internal/CertificateExtensions.cs

**Reply by Kralizek** [reply]
Can you specify which tags and their expected values? I couldn't find anything specific by looking at the file you linked.
